script/FullDeployer.s.sol

```diff
@@ -8,13 +8,49 @@ import {HubDeployer} from "script/HubDeployer.s.sol";
 import {SpokeDeployer} from "script/SpokeDeployer.s.sol";
 
 contract FullDeployer is HubDeployer, SpokeDeployer {
-    function deployFull(uint16 centrifugeId, ISafe adminSafe_, address deployer, bool isTests) public {
-        deployHub(centrifugeId, adminSafe_, deployer, isTests);
-        deploySpoke(centrifugeId, adminSafe_, deployer, isTests);
+    function deployFull(uint16 centrifugeId_, ISafe adminSafe_, address deployer, bool isTests) public {
+        deployHub(centrifugeId_, adminSafe_, deployer, isTests);
+        deploySpoke(centrifugeId_, adminSafe_, deployer, isTests);
+    }
+
+    function run() public {
+        vm.startBroadcast();
+        uint16 centrifugeId;
+        bool isTestnet;
+
+        try vm.envString("NETWORK") returns (string memory network) {
+            string memory configFile = string.concat("env/", network, ".json");
+            string memory config = vm.readFile(configFile);
+            centrifugeId = uint16(vm.parseJsonUint(config, "$.network.centrifugeId"));
+            string memory environment = vm.parseJsonString(config, "$.network.environment");
+            isTestnet = keccak256(bytes(environment)) == keccak256(bytes("testnet"));
+        } catch {
+            console.log("NETWORK environment variable is not set, this must be a mocked test");
+            revert("NETWORK environment variable is required");
+        }
+
+        deployFull(centrifugeId, ISafe(vm.envAddress("ADMIN")), msg.sender, false);
+        // Since `wire()` is not called, separately adding the safe here
+        guardian.file("safe", address(adminSafe));
+        saveDeploymentOutput();
+        vm.stopBroadcast();
     }
 
     function removeFullDeployerAccess(address deployer) public {
-        removeHubDeployerAccess(deployer);
-        removeSpokeDeployerAccess(deployer);
+        bool isTestnet;
+
+        try vm.envString("NETWORK") returns (string memory network) {
+            string memory configFile = string.concat("env/", network, ".json");
+            string memory config = vm.readFile(configFile);
+            string memory environment = vm.parseJsonString(config, "$.network.environment");
+            isTestnet = keccak256(bytes(environment)) == keccak256(bytes("testnet"));
+        } catch {
+            console.log("NETWORK environment variable is not set, this must be a mocked test");
+        }
```

Comment:
Is this need it? Shouldn't be the case IIUC

Reply:
isTestnet is used to determine if .deny() should run or not for a specific deployment, it will be useful once we have mainnet deployments

Reply:
This is to set isTestnet. The above is to set centrifugeID
I could reuse the loading config code but I got lazy 😃

Reply:
But that section is in the removeDeployerAccesss section